dashboard/src/components/AppList/AppListItem.tsx

```diff
@@ -20,7 +20,7 @@ class AppListItem extends React.Component<IAppListItemProps> {
         link={`/apps/ns/${app.namespace}/${app.releaseName}`}
         title={app.releaseName}
         icon={icon}
-        info={app.version || "-"}
+        info={`${app.chart}/v${app.version || "-"}`}
```

Comment:
the / version is confusing, for a second I thought it was repo/name. Maybe we could have a space between these instead?

Reply:
or maybe `colon` which seems more widespread? Alternative I'd go for the version in parenthesis

Reply:
Why do we need `-`? `${app.version || "-"}`

Reply:
I think we should avoid confusing this with the chart version, the Helm CLI uses `{chart.name}-{chart.version}`. IMO a space will be the least confusing.

Reply:
Actually I think that the separation between the two items using just a space is not very visible, let me try using two different lines

Reply:
nvm, I don't like the result of printing them as a column:
<img width="955" alt="screenshot 2018-11-09 at 12 14 17" src="https://user-images.githubusercontent.com/4025665/48259709-14dad080-e419-11e8-95c4-9b64581d516e.png">